deploy/helm/sumologic/values.yaml

```diff
@@ -242,6 +242,16 @@ sumologic:
     collector:
       otelcol:
         enabled: true
+      otelcloudwatch:
+        enabled: false
+        roleArn: ""
+        ## Configure persistence for the cloudwatch collector
+        persistence:
+          enabled: true
+        region: us-east-2
+        pollInterval: 1m
+        # A map of log group and stream prefixes
+        logGroups: {}
```

Comment:
We should provide some examples. We can also provide defaults which will be working according to our documentation

Reply:
```
logGroups:
  fluent-bit:
    names: [fluent-bit]
```
this would be an example. I can add it as a comment instead of a default, as the customer will most likely not have the default log group in cloudwatch which will result in error logs in our collector like "Log group not found"